doc/callbacks.rst

```diff
@@ -388,3 +388,12 @@ In the above examples the callback function is defined in the same scope as the
     solveProblem()
 
 This code can also be found in ``/JuMP/examples/simplelazy2.jl``.
+
+Exiting a callback early
+^^^^^^^^^^^^^^^^^^^^^^^^
+
+If you need to exit the optimization process earlier than a solver otherwise would, it is possible to throw a ``CallbackAbort`` exception in callback code::
+
+    throw(CallbackAbort())
+
+JuMP will take this exception and pass the appropriate error code to the solver so that the environment can be cleaned up. Note that the solver level will most likely throw an exception of its own, e.g. Gurobi will throw a ``GurobiError``.
```

Comment:
http://www.gurobi.com/documentation/5.6/reference-manual/c_grbterminate#routine:GRBterminate

Says it should be GRB_INTERUPTED, we can turn that into something right? Or am I missing something in how this works?